chsdi/views/features.py

```diff
@@ -494,24 +494,19 @@ def _find(request):
 def _format_search_text(columnType, searchText):
     if isinstance(columnType, Text):
         return searchText
-    elif isinstance(columnType, Boolean):
-        if searchText.lower() == 'true':
-            return True
-        elif searchText.lower() == 'false':
-            return False
-        else:
-            raise exc.HTTPBadRequest('Please provide a boolean value (true/false)')
+    elif isinstance(columnType, Boolean):  # pragma: no cover
+        raise exc.HTTPBadRequest('Please do not provide boolean value to the %s' % (columnType))
     elif isinstance(columnType, Integer):
         if searchText.isdigit():
             return int(searchText)
-        else:
+        else:  # pragma: no cover
             raise exc.HTTPBadRequest('Please provide an integer')
     elif isinstance(columnType, Numeric):
         if re.match('^\d+?\.\d+?$', searchText) is not None:
             return float(searchText)
-        else:
+        else:  # pragma: no cover
```

Comment:
this can be checked by providing a non-float when an float is excpected (using find service)